Sources/Extensions/Custom Types/LoginSessionConfiguration+OneLogin.swift

```diff
@@ -4,7 +4,7 @@ extension LoginSessionConfiguration {
     static var oneLoginSessionConfig: LoginSessionConfiguration {
```

Comment:
pet peeve: `sessionConfiguration` is implicit here because it's an extension on `LoginSessionConfiguration`.
It would be _Swiftier_ to have `static var oneLogin: LoginSessionConfiguration` because at the callsite it would be `LoginSessionConfiguration.oneLoginSessionConfig` which is long and tautologous.

Reply:
'tis a good point, will change 🙂